NorthstarDLL/printcommand.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
```

Comment:
Is there a reason the header is called `printcommand.h` (singular) while the C++ source file is called `printcommands` (plural)? ^^

Reply:
should probably be removed if there's nothing in it